src/CoreHandler.php

```diff
@@ -17,10 +16,12 @@ class CoreHandler extends Object implements SelfHandlingCommand
 
     /**
      * Load Stripe private key from .env
+     *
+     * @param string $stripe_priv_key
      */
-    public function init()
+    public function init(string $stripe_priv_key)
     {
-        \Stripe\Stripe::setApiKey(getenv('STRIPE_PRV_KEY'));
+        \Stripe\Stripe::setApiKey($stripe_priv_key);
```

Comment:
Can we `use` this class instead of hardcoding a namespace, here?

What is the difficulty level of passing a stripe object into this method instead? Or maybe into the constructor of this class? This is a great example to show how dependency injection could make testing easier!